### PR TITLE
Changed the welcome message

### PR DESCRIPTION
**Pull request by jadenPete**
"Hello, world!" is better than "Welcome!"...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <title>My Page</title>
 </head>
 <body>
-  <h1>Welcome!</h1>
+  <h1>Hello, world!</h1>
```
Comment (jacobtie) [inline index.html:10]: I don't like this

**Review by jacobtie — CHANGES_REQUESTED** [review]
Good effort

**Review by DylanHalstead — APPROVED** [review]
I like it :(

**Review by jacobtie — APPROVED** [review]
Better. 🚀

**Pull request merged**